lib/mongoid/attributes.rb

```diff
@@ -350,7 +350,7 @@ def lookup_attribute_presence(name, value)
       if localized_fields.has_key?(name)
         value = localized_fields[name].send(:lookup, value)
       end
-      !!value
+      value == true || value.present?
```

Comment:
@estolfo 

Can I use `value.present?` It should have the same effect. All specs pass